controller/volume_controller.go

```diff
@@ -52,6 +52,8 @@ const (
 	VolumeSnapshotsWarningThreshold   = 100
 
 	LastAppliedCronJobSpecAnnotationKeySuffix = "last-applied-cronjob-spec"
+
+	PVCAnnotationHarvesterHciVolumeStatus = "harvesterhci.io/volume-status"
```

Comment:
We should use longhorn namespace instead of customizing for any downstream project.

Reply:
Agree.